src/data/vault.json

```diff
@@ -151,12 +151,14 @@
 			"iconName": "docs",
 			"name": "General Documentation",
 			"path": "docs",
-			"shortName": "Documentation"
+			"shortName": "Documentation",
+			"addOverviewItem": true
```

Comment:
I've updated `vault` and `waypoint` data with newly introduced `addOverviewItem` prop. I've gone the "add" route because a lot of Terraform routes would be duplicative otherwise.

I was admittedly on the fence whether to invert this and add `hideOverviewItem` to the Terraform items, though that felt like it might imply that we're hiding an incoming overview item from `nav-data`, which is not the case. The other thing that swayed me was the idea that adding these "Overview" items is something we should do in content, not in implementation. Intent being that we can deprecate `addOverviewItem` route-by-route post-GA by updating content.

Very open to other approaches here 🙇‍♂️